docker-compose.yml

```diff
@@ -56,9 +56,10 @@ services:
       "DJANGO_SETTINGS_MODULE=settings",
       "watchmedo",
       "auto-restart",
-      "--directory=/data/olympia/src",
+      "--directory=/data/olympia",
```

Comment:
I know `src/` means we don't restart on `settings.py` (or `local_settings.py`) changes, but this was like that before because otherwise there are too many files to monitor as it includes `static/`, `node_modules/`, `storage/` ... (The limit of how many files you can monitor varies depending on the host but we need to keep it as low as possible)

Reply:
Reverted

Reply:
Looks like it's still here ?